SPECS/xapi.spec

```diff
@@ -18,6 +18,7 @@ Provides: gitsha(https://code.citrite.net/rest/archive/latest/projects/XSU/repos
 # XCP-ng patches
 Patch1000: xapi-1.160.1-allow-migrate_send-during-RPU.XCP-ng.patch
 Patch1001: xapi-1.160.1-open-vxlan-port-for-sdn-controller.XCP-ng.patch
+Patch1002: xapi-1.160.2-create-suspended-vm.XCP-ng.patch
```

Comment:
The version we put in the patch name is that of the version of the tarball we created it for, here 1.214.1

Reply:
Oh yeah, my rebase change the version
on it